examples/svelte/src/App.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+    import { router } from './router.js';
+    import { uuid } from './utils.js';
+
+    import Header from './Header.svelte';
+    import Footer from './Footer.svelte';
+    import Item from './Item.svelte';
+
+    import "./app.css";
+    import "todomvc-app-css/index.css";
+    import "todomvc-common/base.css";
+
+    let currentFilter = "all";
+    let items = [];
+    let editing = null;
+
+    function addItem(event) {
+        items.push({
+            id: uuid(),
+            description: event.detail.text,
+            completed: false,
+        });
+        items = items;
+    }
+
+    function removeItem(index) {
+        items.splice(index, 1);
+        items = items;
+    }
+
+    function toggleAllItems(event) {
+        const checked = event.target.checked;
+        items = items.map((item) => ({
+        ...item,
+        completed: checked,
+        }));
+    }
+
+    function removeCompletedItems() {
+        items = items.filter((item) => !item.completed);
+    }
+    
+    router(route => currentFilter = route).init();
```

Comment:
This should really be happening inside an `onMount` callback. Doesn't really matter for this demo since it only runs client-side, but in general we avoid doing anything browser-specific during component initialisation

```suggestion
    onMount(() => {
      router(route => currentFilter = route).init();
    });
```

(needs `onMount` to be imported from `svelte` obvs)